projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -3217,16 +3217,6 @@ class Share_Nextdoor extends Sharing_Source {
 	 */
 	public $icon = '\f10c';
 
-	/**
```

Comment:
Ah nice, I didn't spot that! I copied this from the WhatsApp one. The dangers of copy/paste...